cypress/integration/mediaHomePage/landing_page_current_series_spec.js

```diff
@@ -19,19 +19,19 @@ describe('Testing the Current Series on the Media landing page:', function () {
   });
 
   beforeEach(function () {
-    cy.get('div.media-body').as('featuredSeries')
+    cy.get('body > div:nth-child(9) > section.bg-tan-light.watch-section > div > div:nth-child(2)').as('featuredSeries')
```

Comment:
Not a fan of looking for the nth child. Could you add/use a class or an id to the spot that is being tested